ci/jenkins/test-vm.sh

```diff
@@ -286,8 +286,8 @@ function run_e2e_vms {
 
 function deliver_antrea_vm {
     export_govc_env_var
-    clean_vm_agent ${LIN_HOSTNAMES[@]}
-    clean_vm_agent ${WIN_HOSTNAMES[@]}
+    clean_vm_agent ${LIN_HOSTNAMES[*]}
+    clean_vm_agent ${WIN_HOSTNAMES[*]}
```

Comment:
what's wrong with the original code? I feel the original code is more common.

Reply:
Correct. But other cases when we are passing an array as a parameter we are using * so I thought we should use the same.

Reply:
Please tell Is this okay

Reply:
Could you share a link where we are passing an array as a parameter using `*`?
I feel I only saw `@` in antrea and other code base.
https://github.com/antrea-io/antrea/blob/83fc1d6886fa282f3b09c8af42b669cf48f8b7a2/test/e2e/infra/vagrant/push_antrea.sh#L155

Reply:
Done.